### PR TITLE
Add TCB post: OBIEE part 1: INFODEV-2319

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-63120e9e36/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-ce24de8d50/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-332d9defd1/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-959aff55a6/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-959aff55a6/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-55922eab1d/)

**Comment by RackerKate** [comment]
@cyrichardson For your review.

**Comment by cyrichardson** [comment]
On it, @RackerKate.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-55922eab1d/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-bf6ba6705e/)

**Comment by cyrichardson** [comment]
@RackerKate Suggested changes here: 
[blog_Tips-for-using-Oracle-Business-Intelligence-Enterprise-Edition_cyrEdits.docx](https://github.com/rackerlabs/docs-developer-blog/files/2311973/blog_Tips-for-using-Oracle-Business-Intelligence-Enterprise-Edition_cyrEdits.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-db2a8e85e0/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-8207369637/)

**Comment by RackerKate** [comment]
@catlook for your review.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-407902bb85/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-c1657aa68d/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-089370c2b0/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-e0308497b2/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-36bdf238a8/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-4d0a8e8302/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-c69e3f0629/).

**Comment by RackerKate** [comment]
@catlook For your review.

**Comment by catlook** [comment]
on it

File: _posts/2018-08-23-obiee1.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    entire application to avoid complications due to intra-application
+    dependencies.
+
+    If this attribute configures some other part of the domain such as a
+    server, cluster, or Enterprise JavaBeans (EJB) container, the system can't
+    process the change until you restart the server or cluster.
+
+### Conclusion
+
+Following the steps outlined in this blog post help you fix stuck thread
```
Comment (catlook) [inline _posts/2018-08-23-obiee1.md]: ...  post helps you to fix ...

File: _posts/2018-08-23-obiee1.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+issues and avoid server crashes. You can also configure a Simple Network
+Management Protocol (SNMP) alert that will immediately send you an email alert
```
Comment (catlook) [inline _posts/2018-08-23-obiee1.md]: ... that immediately sends ...

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-7cde7aa2db/)

**Pull request merged**